src/i18n/content/kr/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx

```diff
@@ -13,7 +13,7 @@ Kubernetes는 배포 자동화, 확장 및 컨테이너화된 애플리케이션
 
 [Kubernetes 통합을 실행하고](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/) 나면 [즉시 사용 가능한 Kubernetes 경험을](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer) 통해 Kubernetes에서 실행되는 모든 서비스를 한 곳에서 모니터링할 수 있습니다. 제어 평면 상단부터 단일 파드에서 실행되는 구성에 이르기까지 모든 것을 보여주므로 클러스터 작동 방식의 핵심에 접근할 수 있습니다.
 
-<img title="New Relic - Kubernetes Cluster Explorer" alt="New Relic - Kubernetes Cluster Explorer" src="/images/kubernetes_screenshot-crop_cluster-explorer.webp" />
+<img title="New Relic - Kubernetes cluster explorer" alt="New Relic - Kubernetes cluster explorer" src="/images/kubernetes_screenshot-crop_cluster-explorer.webp" />
```

Comment:
Translation file

Reply:
Thank you @akristen! I know it's a translation file but this change doesn't affect the content itself.